src/main/java/webserver/Router.java

```diff
@@ -58,6 +68,10 @@ public String route(Request request) {
                 return this.HEADRoutes;
             case HTTP_HEADERS.POST:
                 return this.POSTRoutes;
+            case HTTP_HEADERS.PUT:
```

Comment:
There's an opportunity to use polymorphism here.  Namely, you could call something like `method.getRoutes()`.  So the `PUT` method would return `PUTRoutes`, `GET` method would return `GETRoutes`, etc...

This would require converting `String method` to a value type like `class Method {...}` or `enum Method {...}.

That said, I'm not recommending a change here; only pausing for a moment to reflect.  While it's generally said that "switch is not object-oriented", that doesn't mean that all uses of `switch` in a language like Java are inappropriate.  Whether to use `switch` or polymorphism really comes down to these questions

> How many places do I need to modify, when adding a new HTTP method?
> How likely am I to need to add a new HTTP method?

In your case, the HTTP methods are pretty well fixed and unlikely to change and I think this is the only place where I've seen the `switch` logic.  It's not like you have to modify a dozen classes (which would be a violation of the Open-Closed Principle) when adding a new HTTP method.  I forget where I saw this; Robert Martin summarized this really well in one of his books or videos, but I'm not finding a link to it right now.

TL;DR - no change required; just wanted to inspire some reflection.